packages/shared/src/rpc/index.ts

```diff
@@ -180,22 +181,27 @@ export const getIngressEgressEnvironment = createRequest(
   ingressEgressEnvironment,
 );
 
-const rpcAsset = z.union([
+const deprecatedRpcAssetSchema = z.union([
```

Comment:
not sure we even need these anymore

Reply:
We are also not using the `cf_pool_info` anywhere it seems like. Thats the only place where we were using the "deprecated" schemas. I guess thats also safe to remove.